src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20171229_cluster_google_client_id.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="rtitle" id="cluster_add_google_client_id_column">
+        <addColumn tableName="CLUSTER">
+            <column name="googleClientId" type="varchar(254)" />
```

Comment:
I followed the pattern with other columns here, but I wonder why we don't do `varchar(255)` :)

Reply:
I've also seen the length of the varchar not match the length defined in the Slick component!